torchvision/transforms/functional.py

```diff
@@ -187,6 +192,11 @@ def to_pil_image(pic, mode=None):
         if pic.ndim not in {2, 3}:
             raise ValueError('pic should be 2/3 dimensional. Got {} dimensions.'.format(pic.ndim))
 
+        elif pic.ndim == 3:
+            # check number of channels
+            if pic.shape[2] > 4:
```

Comment:
Same here and please use `pic.shape[-1]` instead of `pic.shape[2]`.

Reply:
Is it some convention to use minus-indexing? Does it have some practicality? I am curious

Reply:
Well, it is a sort of convention that the channel is the last index for numpy and 3rd from right for tensors. For this method it does not matter as input tensor or numpy array should be 2 or 3 dims but for other methods accepting tensors with batch dimension, index=0 points to batch dim instead of channel dim.

Reply:
Ok, now I see the point it's practical to count from the right and also for consistency perhaps

Reply:
This code can be removed, right ?